pulumi/folio/postgresql.ts

```diff
@@ -21,31 +20,39 @@ export module deploy {
                 values: {
                     postgresqlPassword: adminPassword
                 }
-            }, { provider: cluster.provider });
+            }, { provider: fd.cluster.provider });
         return instance;
     }
 
-    export function createDatabase(secret: k8s.core.v1.Secret, appNamespace: k8s.core.v1.Namespace) {
+    export function createInClusterDatabase(secret: k8s.core.v1.Secret,
+        appNamespace: k8s.core.v1.Namespace,
+
+        // See https://www.pulumi.com/registry/packages/kubernetes/api-docs/helm/v3/chart/#depend-on-a-chart-resource
+        dependsOn?: Output<CustomResource[]>) {
         return new k8s.batch.v1.Job("create-database", {
             metadata: {
                 name: "create-database",
                 namespace: appNamespace.id,
             },
+
             spec: {
                 template: {
                     spec: {
                         containers: [{
                             name: "create-database",
-                            image: "culibraries/create-database:0.16",
+                            image: "ghcr.io/culibraries/create_database:latest",
```

Comment:
I think we should use specific tags instead of `latest` so that we can customize containers on a per FOLIO version basis and easily run more than one version of FOLIO at a time.